Source/AppAuth/iOS/OIDExternalUserAgentIOS.h

```diff
@@ -34,15 +34,15 @@ NS_ASSUME_NONNULL_BEGIN
 API_UNAVAILABLE(macCatalyst)
 @interface OIDExternalUserAgentIOS : NSObject<OIDExternalUserAgent>
 
-- (nullable instancetype)init API_AVAILABLE(ios(11))
+- (nonnull instancetype)init API_AVAILABLE(ios(11))
```

Comment:
this was giving a warning because NSObject init() is non null. Using nullable has no warnings and makes calling code simpler